manager/controllers/system_packages_export.go

```diff
@@ -70,25 +77,51 @@ func SystemPackagesExportHandler(c *gin.Context) {
 		return
 	}
 
-	data := convertToOutputArray(&loaded)
+	if apiver < 3 {
+		data := buildSystemPackageInlineV2(loaded)
+		OutputExportData(c, data)
+		return
+	}
+	data := buildSystemPackageInlineV3(loaded)
 	OutputExportData(c, data)
 }
 
-func convertToOutputArray(inArr *[]SystemPackageDBLoad) *[]SystemPackageInline {
-	outData := make([]SystemPackageInline, len(*inArr))
-	for i, v := range *inArr {
-		outData[i].SystemPackagesAttrs = v.SystemPackagesAttrs
-		if v.Updates == nil {
-			outData[i].LatestEVRA = v.SystemPackagesAttrs.EVRA
-			continue
-		}
-		var updates []models.PackageUpdate
-		if err := json.Unmarshal(v.Updates, &updates); err != nil {
-			panic(err)
-		}
-		if len(updates) > 0 {
-			outData[i].LatestEVRA = updates[len(updates)-1].EVRA
+func findLatestEVRA(pkg SystemPackageDBLoad) (installable string, applicable string) {
+	installable = pkg.SystemPackagesAttrsV3.EVRA
+	applicable = pkg.SystemPackagesAttrsV3.EVRA
+	if pkg.Updates == nil {
+		return
+	}
+	var updates []models.PackageUpdate
+	if err := json.Unmarshal(pkg.Updates, &updates); err != nil {
+		panic(err)
+	}
+	nUpdates := len(updates)
+	if nUpdates > 0 {
+		applicable = updates[nUpdates-1].EVRA
+		for i := nUpdates - 1; i >= 0; i-- {
+			if updates[i].Status == "Installable" {
+				installable = updates[i].EVRA
+			}
```

Comment:
```suggestion
		for i := nUpdates - 1; i >= 0; i-- {
			if updates[i].Status == "Installable" {
				installable = updates[i].EVRA
				break
			}
```
add break (or return) to not overwrite latest installable EVRA

Reply:
Thanks for spotting the error.